test/unittest/dds/profiles/test_xml_profile.xml

```diff
@@ -288,6 +300,18 @@
                     <affinity>15</affinity>
                     <stack_size>1048576</stack_size>
                 </security_log_thread>
+                <flow_controller_descriptor_list>
+                    <flow_controller_descriptor>
+                        <name>test_flow_controller</name>
```

Comment:
Update this name so the publish mode's flow controller name references an existing flow controller.

```suggestion
                        <name>test_default_flow_controller</name>
```

Reply:
Good catch !